package.json

```
@@ -22,7 +22,7 @@
     "info:prod": "sls info -s prod"
   },
   "engines": {
-    "node": "^8.10.0",
+    "node": ">= v12.0",
```

Please remove `v` character. and use full semver version. See https://github.com/npm/node-semver for details.

```suggestion
    "node": ">= 12.0.0",
```

---

and Please update check-engine dependency ;)